js/navbar/titlebarButtons.js

```diff
@@ -0,0 +1,61 @@
+var titlebarWindowsMinimise = document.getElementById('titlebar-windows-minimise')
+var titlebarWindowsMaximise = document.getElementById('titlebar-windows-maximise')
+var titlebarWindowsRestore = document.getElementById('titlebar-windows-restore')
+var titlebarWindowsClose = document.getElementById('titlebar-windows-close')
+
+titlebarWindowsMinimise.addEventListener('click', function (e) {
+  remote.getCurrentWindow().minimize()
+})
+
+titlebarWindowsMaximise.addEventListener('click', function (e) {
+  remote.getCurrentWindow().maximize()
+})
+
+titlebarWindowsRestore.addEventListener('click', function (e) {
+  if (browserWindow.isFullScreen()) {
+    remote.getCurrentWindow().setFullScreen(false)
+  } else {
+    remote.getCurrentWindow().restore()
+  }
+})
+
+titlebarWindowsClose.addEventListener('click', function (e) {
+  remote.getCurrentWindow().close()
+})
+
+{
+  var browserWindow = remote.getCurrentWindow()
+  switch(process.platform){
+    case 'win32':
+      if (browserWindow.isMaximized()||browserWindow.isFullScreen()) {
+        titlebarWindowsMaximise.hidden = true
+        titlebarWindowsRestore.hidden = false
+      } else {
+        titlebarWindowsMaximise.hidden = false
+        titlebarWindowsRestore.hidden = true
+      }
+
+      browserWindow.on('maximize', function (e) {
```

Comment:
Putting callbacks in the renderer process breaks whenever the browser is reloaded, causing this error:

<img width="562" alt="screen shot 2017-06-18 4" src="https://user-images.githubusercontent.com/10314059/27265041-649a7d66-5453-11e7-8007-8b42fe95ba80.png">

These probably need to be moved to the main process, and send IPC messages to the window.